### PR TITLE
Document using stunnel to test in production mode

### PR DESCRIPTION
**Pull request by openjck**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -14,3 +14,6 @@ locale/**/*.mo
 .tox
 deis
 locale/templates/LC_MESSAGES/django.po
+stunnel.log
+stunnel/*
```
Comment (groovecoder) [inline .gitignore]: Can we change this to `stunnel/stunnel.pem`? Being explicit can help save us from "ARGH! Why is this magically NOT working with this 1 filename?!" moments.

File: stunnel/dev_https [hunk 2]
```diff
@@ -0,0 +1,10 @@
+pid=
+
+cert = stunnel/stunnel.pem
+foreground = yes
+output = stunnel.log
```
Comment (groovecoder) [inline stunnel/dev_https]: And/or change this to `output = stunnel/stunell.log` so all the `stunnel` files are kept (and ignored) together?

File: docs/development.rst [hunk 3]
```diff
@@ -96,10 +96,20 @@ Use production assets
...
+    * Mac: ``brew install stunnel``
+#. Run this from the root directory to generate a local cert and key for
+   stunnel (you can use the default values for all prompts):
+   ``openssl req -new -x509 -days 9999 -nodes -out stunnel/stunnel.pem -keyout stunnel/stunnel.pem``
+#. Run ``stunnel stunnel/dev_https``
+#. In another terminal, run ``HTTPS=1 python manage.py runserver 127.0.0.1:5000``
+#. Go to https://127.0.0.1:8443 and confirm the certificate exception
+#. Browse around the site at https://127.0.0.1:8443
+
+.. _stunnel: https://www.stunnel.org/index.html
```
Comment (groovecoder) [inline docs/development.rst:112]: Can we change the title of this section from "Use production assets" to "Run in production mode" ?

**Comment by openjck** [comment]
Updated

**Pull request merged**